plugins/modded_entity_fabric.js

```diff
@@ -176,7 +260,8 @@ Plugin.register('modded_entity_fabric', {
 	},
 	onunload() {
 		delete Codecs.modded_entity.templates['Fabric 1.14'];
-		delete Codecs.modded_entity.templates['Fabric 1.15+'];
+		delete Codecs.modded_entity.templates['Fabric 1.15-1.16'];

```

Comment:
I think here you may want to do:
```js
		delete Codecs.modded_entity.templates['Fabric 1.15+'];
		delete Codecs.modded_entity.templates['Fabric 1.15-1.16'];
```

because in the case they are upgrading this plugin it's going to hot-swap it, you want to remove the old template added by the old version of the plugin.

Reply:
Updated this in a new commit.

Also added a comment in the code mentioning where the code was inspired from should someone else be utterly confused about what's happening 😂